packages/dds/matrix/src/test/utils.ts

```diff
@@ -222,3 +224,24 @@ export function insertFragmented(
 
 	return matrix;
 }
+/**
+ * Creates a local matrix with the specified size and initial value.
+ */
+export function createLocalMatrix(
```

Comment:
If this is only used in one test file, I would suggest making it a private function in that test file rather than putting it here.

Reply:
It will use for redo/undo test cases